### PR TITLE
[WIP] Refactor Going Live Doc

### PR DESCRIPTION
**Pull request by rachelwhitton**
Closes issue #482 

I need some help getting things squared away. Here is a list of needed content:
- [x] Schedule Backups for Development, Testing, and Production Environments
- [x] Add New Relic Performance Monitoring
- [x] Setup Pingdom alert on non-cached page
- [x] Ensure caching is properly configured for optimized load times
- [x] Enable cacheserver to improve performance for authenticated users
- [x] Site Audit Launch Check:
 - [x]  Run site audit checks for insecure and vulnerable code
 - [x] Check for available plugin/module updates
 - [x]  Detect when too many plugins are installed 
 - [x]  Check for plugins or modules that are known to cause problems
 - [x] Detect large files that should be served by CDN
 - [x] Check database for large tables and correct storage engine
- [x] Load Test common user scenarios
- [x] Verify redirection sends visitors to canonical domain
- [x] Test DNS settings and lower the record TTL prior to host switchover
- [x] Setup redirects for URLs changing. Have plan to monitor and mitigate 404s post-launch to retain search rankings.
- [x] If planning to use SSL, it should be enabled and the cert uploaded.
link to https://pantheon.io/docs/articles/sites/domains/adding-a-ssl-certificate-for-secure-https-communication/
- [x] Normalizing URLs is also important. www / bare domain redirects are oft forgotten at the very least. Leaving the live-* domain in place also has poor SEO implications.

@suzannealdrich can you add to this PR?

### CODE REVIEW

**Comment by rachelwhitton** [comment]
Later todos:

When deploying to live - show this doc 
Add to walkme

**Comment by rachelwhitton** [comment]
@ari-gold @suzannealdrich @JessiFischer here's a screenshot of what the checklist looks like now, the doc has been greatly reduced so that it's easy to get a quick view of what's required. Can ya'll provide feedback/review? (Might be best to check this branch out locally and build sculpin so you can see how the anchors work)

Thanks in advance!


screen shot 2015-06-30 at 4 08 38 pm

File: source/docs/articles/going-live.md [hunk 1]
```diff
@@ -5,204 +5,56 @@ category:
...
+
+Consider enabling [New Relic](/docs/articles/sites/newrelic/new-relic-performance-analysis/) monitoring to non-intrusively track the performance of your site; this is one of the first places Pantheon will look when there are performance concerns.
+
+[Pingodm](https://www.pingdom.com/) can also be used as an additional monitoring service to create site alerts for non-cached pages.
+
+##  Redirects
+
+Setup redirects for possible URL changes as a result of going live. Have plan to monitor and mitigate 404 Page not found errors post-launch to retain search rankings. For more information, see [Redirecting Incoming Requests](/docs/articles/sites/code/redirect-incoming-requests/)
+<div class="alert alert-danger" role="alert">
+<strong>Warning</strong>: For best results, do <strong>not</strong> rely global redirect logic within the application configuration (e.g. wp-config.php or settings.php). The <a href="https://www.drupal.org/project/redirect" target="blank">Redirect module</a> is recommended. </div>
```
Comment (rachelwhitton) [inline source/docs/articles/going-live.md]: @suzannealdrich is this what you meant by global redirects? Are there any WP redirect plugins we can recommend? Did I leave anything out here? @ari @JessiFischer

**Comment by rachelwhitton** [comment]
Added **See Also** section:
screen shot 2015-07-01 at 10 11 45 am

Should there be any other docs included in this section?

**Comment by rachelwhitton** [comment]
[WordPress Hosting](https://pantheon.io/features/wordpress-hosting) | [Drupal Hosting](https://pantheon.io/features/drupal-hosting) | [Website Management Platform](https://pantheon.io/platform/website-management-platform)


@nataliejeremy they are links ^^

File: source/docs/articles/going-live.md (hunk 1 above)
Comment (ari) [inline source/docs/articles/going-live.md]: Please stop inviting me to your tasks.

**Comment by rachelwhitton** [comment]
Screenshot:
screen shot 2015-07-07 at 1 53 28 pm


Verbal +1's given from @JessiFischer and @suzannealdrich 

Additional feedback on the way from @ari-gold 

@nataliejeremy can you review for grammar/style?

File: source/docs/articles/going-live.md [hunk 2]
```diff
@@ -5,204 +5,56 @@ category:
...
-		</tr>
-	</tbody>
-	</table>
+## Best Practices for Launching Sites Checklist
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Deploy to the Live Environment](/docs/articles/sites/code/using-the-pantheon-workflow/#3.-deploy-code-to-live)<br>
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Select a Plan for Your Site](/docs/articles/sites/settings/selecting-a-plan/)<br>
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Schedule Daily and Weekly Backups](#schedule-backups)<br>
+**Configure Site Monitoring Services:**<br>
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Enable New Relic for Performance Analysis](#monitoring-services)<br>
+&nbsp;&nbsp;&nbsp;<span class="glyphicon  glyphicon-unchecked" aria-hidden="true"></span> [Create Site Alerts with Pingdom](#monitoring-services) (Optional)<br>
```
Comment (ari-gold) [inline source/docs/articles/going-live.md]: @JessiFischer  @suzannealdrich  @erik-pantheon Wouldn't you want to do this after updating DNS?

**Comment by ari-gold** [comment]
:+1:

**Pull request merged**